### PR TITLE
Support for Fastify v3

### PR DESCRIPTION
**Pull request by delvedor**
As titled.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.test-d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import fastifySensible from '.'
+
+const app = fastify()
+
+app.register(fastifySensible, {
+  errorHandler: true
+})
+
+app.get('/', (req, reply) => {
+  reply.notFound()
```
Comment (Ethan-Arrowood) [inline index.test-d.ts:11]: You should use `tsd` and expect types in this test file

File: package.json [hunk 2]
```diff
@@ -23,18 +23,22 @@
...
+    "fastify": "^3.0.0-alpha.1",
+    "standard": "^14.3.3",
+    "tap": "^14.10.7",
+    "tsd": "^0.11.0"
   },
   "dependencies": {
-    "@types/node": "^12.0.7",
-    "fast-deep-equal": "^2.0.1",
-    "fastify-plugin": "^1.5.0",
+    "@types/node": "^13.13.2",
```
Comment (mcollina) [inline package.json:32]: I would not put this in dependencies. It should be a devDep.

File: package.json [hunk 3]
```diff
@@ -23,18 +23,22 @@
...
+    "fast-deep-equal": "^3.1.1",
+    "fastify-plugin": "^2.0.0",
     "forwarded": "^0.1.2",
-    "http-errors": "^1.7.2",
-    "proxy-addr": "^2.0.5",
+    "http-errors": "^1.7.3",
+    "proxy-addr": "^2.0.6",
```
Comment (mcollina) [inline package.json:37]: Do we still need proxy-addr? I think this feature is built in Fastify now..?

File: package.json [hunk 4]
```diff
@@ -23,18 +23,22 @@
   },
   "homepage": "https://github.com/fastify/fastify-sensible#readme",
   "devDependencies": {
-    "fastify": "^2.3.0",
-    "standard": "^12.0.1",
-    "tap": "^12.7.0"
+    "fastify": "^3.0.0-alpha.1",
```
Comment (Ethan-Arrowood) [inline package.json:26]: use `rc` instead of `alpha`

**Pull request merged**